blogSignUp.php

```diff
@@ -0,0 +1,53 @@
+<?php
+include ('include/include_all.php');
+echo "<link rel='stylesheet' href='maindesign.css' />";
+
+$Errorz = array();
+
+function ValidateSignUp(){
+	global $Errorz;
+	if (trim($_REQUEST['UName'])==''){
+		$Errorz['UName']="required";
+	}
+	if (trim($_REQUEST['PWord'])==''){
+		$Errorz['PWord']= "required";
+	}
+}
+
+function SignUp($Username, $Password){
+	$result = dbQuery("
+		INSERT INTO BlogLogIns(Username, Password)
+		VALUES('$Username', '$Password')
+	")->fetchall();
+	return $result;
+}
+
+
+function ConfirmMessage(){
+	global $Errorz;
+	if(isset($_REQUEST['submitSignUp'])){
+	ValidateSignUp();
+		if(sizeof($Errorz) == 0){
+			SignUp(($_REQUEST['UName']),($_REQUEST['PWord']));
+			echo "<p class='ConfirmMsg'>Nice, now you're a member of Lindsay's fanclub. Click <a href = 'http://localhost:8888/blogLogin.php'>here</a> to log in.</p>";
```

Comment:
After submitting a form, you always want to redirect the page (even if it's back to this same page). Otherwise, reloading the page would re-submit the form.